test/openAPI/features/support/helpers/helpers.js

```diff
@@ -369,4 +369,24 @@ module.exports = {
     },
     required: ['ID', 'FirstName', 'LastName', 'BirthCertificateID'],
   },
+  replaceKeyWithValueFromJson,
+  readJsonFile,
 };
+
+const fs = require('fs');
+let jsonData = readJsonFile('testCustomParameters.json') || readJsonFile('testDefaultParameters.json');
+
+function replaceKeyWithValueFromJson(key) {
+  let processedKey = key.replace('${', '').replace('}', '');
+  return jsonData[processedKey] || key;
```

Comment:
I think that when processedKey doesn't exist then exeption could be raised. If for any reason json schema would be invalid then it'd be hard to track the issue.

Reply:
okay, if we don't want to keep any hard-coded data we can change also to return: only `jsonData[processedKey]`